code/parse/sexp.cpp

```diff
@@ -38736,9 +38735,7 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"\t2:\tA short string which will appear in the warning, or a message if the string matches a message name.\r\n"},
 
 	{ OP_GRANT_PROMOTION, "Grant promotion (Action operator)\r\n"
-		"\tIn a single player game, this function grants a player an automatic promotion to the "
-		"next rank which the player can obtain.  If he is already at the highest rank, this "
-		"operator has no effect.  It takes no arguments." },
+		"\tIn a single player game, this function grants a player an automatic promotion to the next rank which the player can obtain. If they are already at the highest rank, this operator has no effect. It takes no arguments." },
```

Comment:
Even if it isn't necessarily consistent among the descriptions, keep the line breaks this description had. Ditto with set-subspace-drive below.

Reply:
Done. May I ask what the pattern is supposed to be for when I write operators in the future?

Reply:
The idea is that on smaller screens, it can be cumbersome to read the source code if a get line gets too long, I forget what the actual character number is, I know visual studio has a feature where till draw a little line on screen so someone knows not to write a line longer than that.

But overall, its a very minor concern, as mentioned, FSO isn't exactly consistent about it, and screens are pretty big these days, but its still a little easier to read.